.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker Image CI
+
+on:
+  release:
+    types: [published]
+    
+env:
+  IMAGE_NAME: build
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+  
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag image
+
+    - name: Log into registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
+
+    - name: Push image
+      run: |
+        IMAGE_ID=docker.pkg.github.com/${{ github.repository }}/$IMAGE_NAME
+        
+        # Strip git ref prefix from version
+        VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+        
+        # Strip "v" prefix from tag name
+        [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
+        
+        # Use Docker `latest` tag convention
+        [ "$VERSION" == "master" ] && VERSION=latest
```

Comment:
Remind me, why would we do this? And if we upload the next image as `latest`, how do we refer to it as `1.1.0` afterwards?

Reply:
I borrowed this template from GitHub Actions, and left some of the 'compatibility' code in, incase we wanted to do non-release CI ever. This just calls it 'latest' if it's a build from `master` (as opposed to a release) - that may have been triggered by a merge/nightly.

We could strip it (and some other compatibility  code, eg: `v` stripping) that we're not using right now.

Reply:
Yes, please strip. It's misleading to have code in here which we don't expect to do anything.

Reply:
Removed in https://github.com/GafferHQ/build/pull/16/commits/66b73e8b3b5683164965643e61a4604745637e99